### DISCUSSION TITLE
Is it possible to use Code Maid to automatically change String to string?

### DISCUSSION THREAD

**Discussion by jobrien9** [codecadwallader/codemaid#993] [Q&A]
Hey all,

I use Code Maid for C#. It's a great tool, and I appreciate the work y'all have put into it! 

My team is inconsistent when using Strings for variable types. Some places of the code use String with an capital 'S', and others use the lowercase "string". Is it possible to configure Code Maid to automatically detect this and enforce consistency? 

Thanks!

**Comment by codecadwallader** [comment]
Thanks for the suggestion!  We do not currently have functionality that handles that update.
